packages/genesys-spark-tokens/data/ui/data_display.json

```diff
@@ -993,12 +993,6 @@
       }
     }
   },
-  "global_nav": {
-    "side_menu": {
-      "value": "linear-gradient(135deg, {color.emerald.500} 0%, {color.emerald.200} 100%)",
-      "type": "color"
-    }
-  },
```

Comment:
Confirmed with @samlimby-genesys that this was a bug not a wanted token.
This token was blocking other wanted tokens from being created that is why I deleted instead of deprecating.